integrations/dbt/bigquery/dbt/adapters/openlineage_bigquery/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.15.2'
```

Comment:
We'll want to avoid using the `version.py`. Mind if we follow the same versioning configuration as [marquez-airflow](https://github.com/MarquezProject/marquez/tree/main/integrations/airflow) and [marquez-python](https://github.com/MarquezProject/marquez/tree/main/clients/python)?

Reply:
This would introduce circular dependency. In `__init__.py` we need to have `AdapterPlugin` implementation, which needs `ConnectionManager`. And we need version for `ConnectionManager`, since it uses it in producer string.

Reply:
I see. I can try a few things to get versioning to work, but let's merge! I can open a separate PR for the versioning fix.